build.rs

```diff
@@ -0,0 +1,68 @@
+extern crate glob;
+
+use std::fs::File;
+use std::io::Write;
+use std::io::prelude::*;
+use std::process::{Command, Stdio};
+use glob::glob;
+
+fn main() {
+    println!("cargo:rerun-if-changed=\"src/protos/*.proto\"");
+
+    generate_pbs();
+    generate_protos();
+}
+
+fn generate_pbs() {
+    for entry in glob("./games/*.pbtxt").expect("Failed to read glob pattern") {
+        match entry {
+            Ok(path) => generate_pb(path),
+            Err(e) => println!("{:?}", e),
+        }
+    }
+}
+
+fn generate_pb(path: std::path::PathBuf) {
+    let file_name = &path.file_stem().unwrap().to_str().unwrap();
+    let output_file = File::create(format!("games/{}.pb", &file_name)).unwrap();
+
+    let mut input_file = File::open(&path).expect("Failed to open file");
+    let mut contents = String::new();
+    input_file
+        .read_to_string(&mut contents)
+        .expect("Failed to read from file.");
+
+    let mut protos_cmd = Command::new("protoc")
+        .arg("--encode=Maeve.Game")
+        .arg("protos/master.proto")
+        .stdin(Stdio::piped())
+        .stdout(Stdio::from(output_file))
+        .spawn()
+        .expect("Failed to compile pb file.");
+
+    /* Assign _ because it must be used */
+    let _ = protos_cmd
```

Comment:
Can you do ".expect()"?

Reply:
Added an [expect()](https://github.com/AnotherGroupChat/maeve/pull/36/commits/e544f582155b255da0acaa441a2515410ad21c3e#diff-a7b0a2dee0126cddf994326e705a91eaR49)

Will update the readme tomorrow